paymentsheet/src/main/java/com/stripe/android/paymentsheet/specification/Spec.kt

```diff
@@ -1,22 +1,22 @@
-package com.stripe.android.paymentsheet.forms
+package com.stripe.android.paymentsheet.specification
```

Comment:
nit: all other packages are plural, probably change this to `specifications` or `specs` too?